### PR TITLE
Cloud docs: Organization, SSO

### LINKED ISSUE #1649: Cloud docs: Organization management
### Related code PR

_No response_

### Which part(s) of the docs might be affected or should be updated? And how?

Adding concept and how-to guides for the new organization management feature.

### Reference

_No response_

### LINKED ISSUE #1650: Cloud docs: SSO setup and login
### Related code PR

_No response_

### Which part(s) of the docs might be affected or should be updated? And how?

Adding how-to guides for SSO feature.

### Reference

_No response_

### PR DESCRIPTION
**Pull request by CharlieSYH**
## Info

- **Description**

  - [ What's changed? Which parts of the docs are affected? ]

- **Notes**

  - Hiding content related to service accounts for now.

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1649
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1650

## For reviewers

- **Preview**

  - https://pr-1662.d2fbku9n2b6wde.amplifyapp.com/cloud/organization-overview/

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: cloud/images/org-architecture.png
Comment (wjf3121) [inline cloud/images/org-architecture.png:1]: all clusters shares one VPC we pre-provisioned so an org doesn't have the ability to provision VPCs

File: cloud/organization-service-account.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+3. Enter a descriptive name for your service account and write a short description.
+  
+4. Click **Confirm**. 
+
+   <img
+   src={require('./images/org-service-account.png').default}
+   alt="Service account list"
+   />
+
+## Generate an API key
```
Comment (wjf3121) [inline cloud/organization-service-account.md:35]: Note that API key is not yet able to be used until this PR is checked in: https://github.com/risingwavelabs/risingwave-cloud/pull/5134

**Review by wyhyhyhyh — APPROVED** [review]
Looks good to me. We can upload this first.

**Review by WanYixian — APPROVED** [review]
LGTM!

File: cloud/organization-overview.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+slug: /organization-overview
+---
+
+## Concept
+
+In RisingWave Cloud, an organization serves as the central entity that owns all resources, such as accounts, clusters, database objects, and VPC connections. Each organization provides all associated users with access to the same set of resources for seamless collaboration among them. When you sign up for RisingWave Cloud, an organization is automatically created for you. 
+
+In an organization, you can:
+
+- Share clusters, databases (and their tables, sources, sinks, and views), VPC connection, and other resources with all users.
```
Comment (wjf3121) [inline cloud/organization-overview.md]: nit: s/VPC connection/VPC connections ?

File: cloud/images/org-architecture.png
Comment (wjf3121) [reply cloud/images/org-architecture.png:1]: Unresolve the comment, seems not changed?

**Comment by wjf3121** [comment]
cc @xuhui-lu to take a look

File: cloud/images/org-architecture.png
Comment (CharlieSYH) [reply cloud/images/org-architecture.png:1]: I removed this diagram from the docs. Guess I forgot to delete the file.

**Review by wjf3121 — APPROVED** [review]
Looking good!

**Review by ShanlanLi — APPROVED** [review]
LGTM

**Comment by CharlieSYH** [comment]
Merging this PR in an hour. Please have a quick look, @xuhui-lu.

**Pull request merged**